src/main/java/org/opentripplanner/graph_builder/module/time/Cluster.java

```diff
@@ -0,0 +1,45 @@
+package org.opentripplanner.graph_builder.module.time;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Cluster implements Comparable<Cluster> {
+
+    private int id;
+    private TimeTable[] timetable;
+    private EdgeData[] edges;
+
+    public int getid() {
+        return id;
+    }
+
+    public void setid(int id) {
+        this.id = id;
+    }
+
+    public TimeTable[] gettimetable() {
+        return timetable;
+    }
+    public ArrayList<TimeTable>gettimetableas(){
+    ArrayList<TimeTable> s =new ArrayList<> ( Arrays.asList(timetable));
```

Comment:
Tutaj wystarczy `return new ArrayList<>(Arrays.asList(timetable));`. Btw nazwa metody to powinno być `getTimetableAsArrayList` czy coś takiego.